src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/ReplayHeuristic.java

```diff
@@ -70,6 +70,10 @@ public int chooseLiteral() {
 			return DEFAULT_CHOICE_LITERAL;
 		}
 		int replayChoiceSignedAtom = choicesIterator.next();
+		if (replayChoiceSignedAtom == 0 ) {
+			// Use 0 to signal no more choices.
+			return DEFAULT_CHOICE_LITERAL;
+		}
 		int replayChoiceAtom = Math.abs(replayChoiceSignedAtom);
 		int replayChoiceLiteral = Literals.atomToLiteral(replayChoiceAtom, replayChoiceSignedAtom > 0);
```

Comment:
I think that then the lines below might trigger an exception, because `DEFAULT_CHOICE_LITERAL` is not an active choice atom. Also, I like that now the value of the 0 from the command-line argument is independent from the actual value of `DEFAULT_CHOICE_LITERAL`.

Reply:
I agree.